index.js

```diff
@@ -80,6 +81,29 @@ const Home = () => {
         // Set the new state
         setFile({ url, meta, pkg, code });
         replaceState(`?${url.replace('https://unpkg.com/', '')}`);
+        try {
+          var parser = new DOMParser();
```

Comment:
LGTM, especially since @lukejacksonn preferred it. No idea how performant or not this might be though. Had a quick try on perf.link but couldn't get it to run, even with some modifications

Reply:
Resolving this, but storing the alternative solution below for just in case we ever peek back at this PR (v unlikely):

```js
const getPackageVersions = pkgName => (
  fetch(`https://unpkg.com/${pkgName}/`)
    .then(res => res.text())
    .then(res => JSON.parse(res.match(/"availableVersions":(\[.*?\])/)[1]))
)

getPackageVersions("lodash-es").then(console.log);
getPackageVersions("react@16.8.6").then(console.log); // doesn't seem to play nicely without version number sometimes
```